stdlib/public/core/HashedCollections.swift.gyb

```diff
@@ -2786,6 +2840,14 @@ final internal class _Native${Self}Buffer<${TypeParametersDecl}> :
 #endif
 }
 
+
+/// This type provides most of the actual functionality for Dictionary,
```

Comment:
Sentence fragment please, e.g. "A wrapper around a RawNative${Self}Buffer that provides most of the functionality for ${Self}"